make/test.mk

```diff
@@ -116,15 +124,33 @@ test-e2e-registration-local:
 .PHONY: e2e-run-parallel
 e2e-run-parallel:
 	@echo "Running e2e tests in parallel..."
+	oc patch -n ${HOST_NS} toolchainconfigs.toolchain.dev.openshift.com config --patch='{"spec":{"global":{"publicViewer":{"enabled":false}}}}' --type=merge
+	oc delete -n ${HOST_NS} pods -l control-plane=controller-manager
+	oc delete -n ${HOST_NS} pod -l name=registration-service
```

Comment:
Nit: What's the difference between `oc delete pods` (plural) and `oc delete pod` (singular)?  I'd prefer a convention that we stick to.

Reply:
a typo 😄 good catch!